### DISCUSSION TITLE
How can I Automatic Class Sorting in Twind?

### DISCUSSION THREAD

**Discussion by Steffany-Martins** [tw-in-js/twind#395] [Q&A] [answered]
I found this article about Prettier, and this pluggin was used to sort classes. How can I do the same in Twind? 
https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

**Comment by rschristian** [comment, marked as answer]
You can use that very plugin so long as you're using `class` with Twind (like you would with Tailwind).

If you're using the `tw` tagged template you may need to fork & modify the plugin to get it to correctly pick up on that.

**Reply by Steffany-Martins** [reply]
Thank you!
